Bottom: 51cfb338f452fa7f5817bcfeb086a3efcf3c730d
Top:    f135b54236376bb9b9fda7b762c5ed16cff4ae26
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 18:11:20 +0000

Refresh of convert-argument-in-hid_flags-

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 57713b1..60b5780 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -138,12 +138,12 @@ static int flip_x = 0, flip_y = 0;
 static int autofade = 0;
 
 static int
-flag_flipx (void *)
+flag_flipx (void *data)
 {
   return flip_x;
 }
 static int
-flag_flipy (void *)
+flag_flipy (void *data)
 {
   return flip_y;
 }
